source/fundamentals/logging.txt

```diff
@@ -62,7 +62,8 @@ logs all debug messages to the console:
      b.SetMinimumLevel(LogLevel.Debug);
    });
  
-   var settings = MongoClientSettings.FromConnectionString("<your connection string>");
+   var settings = MongoClientSettings.FromConnectionString("<your
+   MongoDB URI>");
```

Comment:
I: make sure you revert the line breaks that your formatter creates when it comes to code examples.

```suggestion
   var settings = MongoClientSettings.FromConnectionString("<your MongoDB URI>");
```